### ISSUE TITLE
Rule idea: no-tabs

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1781]
Why? I don't see tab anywhere in my code:
Valid, but i wan't to invalid:
```css
/*	Some comment with tabs	*/
a::before {
  content: "	Opts here tab:(";
}
```
Or in nonstandard syntax(SCSS)
```scss
.#{$class-selector}			a { // I really do not like the tab
    color: #fff;
}
```

**Comment by jeddy3** [comment]
```scss
.#{$class-selector}			a { // I really do not like the tab
    color: #fff;
}
```

Is that a dupe of https://github.com/stylelint/stylelint/issues/1592?

So, it's tabs within comments and content strings you wish to disallow? I'm wary of adding a wide-reaching rule to do this, one that could conflict with things like https://github.com/stylelint/stylelint/issues/1592 or `indentation`.

Is there another approach here? Smaller, more focused rules perhaps? Or we could push this to a plugin?

**Comment by evilebottnawi** [comment]
@jeddy3 no, this is non-standard syntax, we ignore this selectors

**Comment by evilebottnawi** [comment]
@jeddy3 It is also possible we'll meet another place where there is no check whitespaces, `no-tabs` be good

**Comment by jeddy3** [comment]
Any objections if I label this as a plugin. Like [the eslint issue](https://github.com/eslint/eslint/issues/3521#issuecomment-135182643) I think this too broad for a core rule and doesn't adhere to our ["no overlapping functionality" criteria](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#let-everyone-know).

**Comment by evilebottnawi** [comment]
@jeddy3 maybe good rule for `file-system-lint`
